scripts/alluxio/alluxio-systemd.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
```

Comment:
remove 2021

Reply:
Deleted this file, and will add it to docker image.